dhis2/dhis2-core
# feat: GeoJSON import - validate and reduce coordinates to pairs [DHIS2-14634]

### Summary
Validation of `coordinates`. Empty coordinates should result in validation error. Empty is `null`, `[]` or any level of array that in the end only contains `null`, `[]` as innermost element.

In addition the `coordinates` array is reduced to long/lat pairs in case there is a 3rd (or more) components, for example:

    [1]                  => [1]
    [1,2]                => [1,2]
    [1,2,3]              => [1,2]
    [[1,2,3],[4,5,6]]    => [[1,2],[4,5]]

### Automatic Testing
New unit test was added for low level coordinate handling in `CoordinateUtils`.
A controller test scenario was added for the validation and for the coordinate pair reduction.

### Manual Testing
* Import the example file from the ticket (not sure the target organisation units exist)
* Check the coordinates on the target and see that one long/lat was retained
* Check that the last entry with empty coordinates caused a validation error during import
